frontend/src/App.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import './App.css';
+import Form from './components/Form/Form';
+import CategoryForm from './components/CategoryForm/CategoryForm';
+import ToDoList from './components/ToDoList/ToDoList';
+import { Category, ItemList } from './types/common';
+import Categories from './components/Categories/Categories';
+import { todoService } from './services/todoService';
+
+function App() {
+  const [allLists, setAllList] = useState<Category[]>([]);
+
+  const [selectedListIndex, setSelectedListIndex] = useState(0);
+
+  const getAllLists = async () => {
+    const response = await todoService.getAllLists();
+    setAllList(response);
+  };
+
+  useEffect(() => {
+    void getAllLists();
+  }, []);
+
+  const updateSelectedList = (id: string) => {
```

Comment:
name of the function doesn't represent what it actually do

Reply:
remaned